back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -171,6 +171,9 @@ def create
     input.creation_phase = (phase_for_input if !phase_for_input.pmethod.transitive?)
     input.phase_ids = [phase_for_input.id] if params.dig(:idea, :phase_ids).blank?
 
+    # Non persisted attribute needed by policy & input for 'everyone' participation only
+    input.request = request if phase_for_input.pmethod.everyone_tracking_enabled?
+
```

Comment:
Tried using `policy_context` but this is needed by the model as well as the policy

Reply:
I've updated the comment to make it clearer - it's used in the anonymous participation concern